base_layer/wallet_ffi/src/callback_handler.rs

```diff
@@ -96,7 +96,7 @@ where TBackend: TransactionBackend + 'static
         callback_transaction_mined: unsafe extern "C" fn(*mut CompletedTransaction),
         callback_direct_send_result: unsafe extern "C" fn(TxId, bool),
         callback_store_and_forward_send_result: unsafe extern "C" fn(TxId, bool),
-        callback_transaction_cancellation: unsafe extern "C" fn(TxId),
+        callback_transaction_cancellation: unsafe extern "C" fn(*mut CompletedTransaction),
```

Comment:
The header doesn't match the wallet create for this.
```
 void (*callback_transaction_cancellation)(unsigned long long)
```

needs to be changed to:
```
void (*callback_transaction_cancellation)(struct TariCompletedTransaction*),
```